app/models/follow.rb

```diff
@@ -0,0 +1,7 @@
+class Follow < ActiveRecord::Base
+  has_many :followers, class_name: 'User', primary_key: :follower_id, foreign_key: :id
+  has_many :following, class_name: 'User', primary_key: :following_id, foreign_key: :id
```

Comment:
Line is too long. [87/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.